018_go_list_mod_graph_why/README.md

```diff
@@ -0,0 +1,459 @@
+<!-- __JSON: gobin -m -run myitcv.io/cmd/egrunner script.sh # LONG ONLINE
+
+### Introduction
+
+`go list`, `go mod why` and `go mod graph` can be used to analyse modules and packages. In this guide we introduce each
+command as we analyse the `{{PrintOut "main module"}}` module:
+
+{{PrintOut "base repo"}}
```

Comment:
This renders as:

```
as we analyse the github.com/go-modules-by-example-staging/listmodwhygraph module:

https://github.com/go-modules-by-example-staging/listmodwhygraph
```

I'd prefer to drop the https link, and simply make the main module path contain a link to the repository. The two are the same URLs, anyway.

Reply:
Done.